wiki/osu!wiki_Contribution_Guide/Locally/en.md

```diff
@@ -0,0 +1,71 @@
+# Locally
+
+*Main page: [osu!wiki Contribution Guide](/wiki/owcg).*
+
+*This article continues from the main page* and assumes that you will be using [GitHub Desktop](https://desktop.github.com). You may use other git clients on your own terms. Even if you do work locally, you will still need to access GitHub to create pull requests to make your changes happen.
+
+## Installing GitHub Desktop
+
+You will need to then install [GitHub Desktop](https://desktop.github.com). See [Getting Started with GitHub Desktop](https://help.github.com/desktop-classic/guides/getting-started/) for help.
+
+When the installer is completed, GitHub Desktop will prompt you to login. Click on the `Sign into GitHub.com` link and enter your GitHub's username (or the email you used) and password. You will then be prompted to configure Git, just fill in the prompts and continue. On the next screen, it will prompt you if you want to submit anonymous usage data, it is your call if you want to opt-out or not.
+
+## Clone
+
+Once you are done setting up GitHub desktop, click on `Clone a repository`.
+
+![](img/github-desktop-clone.jpg)
+
+This will show a list of your repos; select the `osu-wiki` repo. The location of the cloned repo is up to you, but take note as to where you clone it to.
+
+![](img/github-desktop-select-repo.jpg)
+
+Once you have decided on the clone location, click `Clone`.
+
+## Branching
+
+This step is technically *optional*, but is heavily recommended due to reasons that you will need to look up for yourself.
+
+Click on the `Current branch` dropdown located at the top of the menu and enter a name for this branch. It can be anything but it could (and probably should) relate to the article you are changing.
```

Comment:
add some suggestion about branch naming ?

Reply:
![image](https://user-images.githubusercontent.com/14264992/32686867-209c2900-c674-11e7-8890-93c086472b4f.png)

from what Repflez had answered, I think that `It can be anything but it could (and probably should) relate to the article you are changing.` should be enough